src/domain/products/batch-job/import.tsx

```diff
@@ -141,6 +141,7 @@ function ImportProducts(props: ImportProductsProps) {
         deleteFile({ file_key: fileKey })
       }
       cancelBathJob()
+      setBatchJobId(undefined)
     } catch (e) {
       console.log(e)
```

Comment:
nit: Can we do some better error handling in this case? Generally don't think that we should be outputing errors to the console, at least not in production.

Reply:
agree - I removed the log and added notifications in case requests fail